package.json

```
@@ -18,6 +18,12 @@
     "LICENSE"
   ],
   "main": "dist/index.js",
+  "exports": {
+    "./lib": {
+      "types": "./dist/lib/export.d.ts",
+      "import": "./dist/lib/export.js"
```

2/5 

Seems vaguely odd to point `package-name/lib` imports to `lib/export.js`. Usually you'd point directories at an `index.js` or something. Maybe make it this?

```suggestion
    "./lib/export": {
      "types": "./dist/lib/export.d.ts",
      "import": "./dist/lib/export.js"
```

??

---

renamed to index